ja_JP/docs/recipes/code-coverage.md

```diff
@@ -56,24 +88,68 @@ ES5をカバーするには、単純にテストスクリプトの先頭に`nyc`
 	}
 }
 ```
+> 注意: `BABEL_ENV=production` はWindowsでは使えないので、`set`を使わなきゃなりません。(`set BABEL_ENV=production`). クロスプラットフォームビルドが必要であれば、[`cross-env`]を確認してください。
 
-ES6をカバーするには、単純にテストスクリプトの先頭に`nyc`と`--babel`フラグを追加します。このnpmスクリプトがコードカバレッジとテストを扱います。
+これはAVAにはあまり影響しない部分であり、Babelの`env`からAVAに対応できるように設定を操作する例です。
+
+### Babelのrequire hook
+
+Babelのrequire hookを使うには、`package.json`のAVAの設定の`require`に`babel-core/register`を追加してください。
 
 ```json
 {
-	"scripts": {
-		"test": "nyc --babel --reporter=text ava"
+	"ava": {
+		"require": ["babel-core/register"]
 	}
 }
 ```
 
+*参考*: コマンドラインからrequire hookを設定することもできます: `ava --require=babel-core/register`。しかし、`package.json`に入れると毎回フラグを追加しなくていいです。
```

Comment:
`参考` should be replaced with `注記`.

```
しかし、`package.json`に入れると毎回フラグを追加しなくていいです。
```
I think it is a little bit off. It would be better like below.
```
けれども`package.json`に設定することで、このフラグを繰り返しタイプすることを防ぐことが出来ます。
```